### DISCUSSION TITLE
Need a TLS Flag to be added for Redis

### DISCUSSION THREAD

**Discussion by chriscrossman-ibm** [directus/directus#7723] [Feature Requests]
Hi there, my team is using Directus and we recent migrated to AWS and are using AWS ElastiCache (Redis) which is looking for a TLS flag to be sent. Looking in the documentation I was not able to find any environment variable for sending TLS flag for Redis. Have you encountered this problem before? If not, is there any way we could get Redis TLS flag environment variable added? 

NodeJS adding TLS: https://docs.upstash.com/howto/connectwithtls (for our other NodeJS services, we followed this and passed blank object for tls)
AWS ElastiCache Redis with TLS: https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/in-transit-encryption.html

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/issues/7724#issuecomment-909350406

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
